cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -1500,7 +1501,7 @@ def get_state(self):
 		if self.client_mode != "": # not processing an actionlib client
 			return self.client_state
 		elif self.client == None:
-			return None
+			return GoalStatus.LOST
```

Comment:
in what context did you have to change this?

Reply:
in a test script we're calling `sss.get_state()` for a non initialized action client and check for a valid actionlib GoalStatus. So returning a valid actionlib status (`LOST`) is better then returning `None`.